src/main/java/org/mastodon/io/yaml/WorkaroundConstructor.java

```diff
@@ -31,15 +31,17 @@
 import java.util.List;
 import java.util.Map;
 
+import org.yaml.snakeyaml.LoaderOptions;
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.nodes.MappingNode;
 import org.yaml.snakeyaml.nodes.SequenceNode;
 
 public class WorkaroundConstructor extends Constructor
 {
-	public WorkaroundConstructor( final Class< ? > theRoot )
+
+	public WorkaroundConstructor( LoaderOptions loadingConfig )
```

Comment:
There is a `Constructor` constructor which takes `theRoot` and `loadingConfig`. Probably better to use that?